### PR TITLE
Added ability to get ticket link

### PR DESCRIPTION
**Pull request by DSofter**
## Content
Added ability to get ticket link. The ticket link is generated from the discount code from ti.to.

Free feel to try it out!

## Reviewers
@dunphyben @prashanthr

### CODE REVIEW

File: src/components/coding-challenge/index.js [hunk 1]
```diff
@@ -53,7 +57,23 @@ export default class CodingChallenge extends Component {
...
+          isFetchingTicket: false,
+          ticketLink: `https://ti.to/sida/frontiersf-2018/discount/${discountCode}`,
+        })
+      }
+    })
+    .catch((error) => {
+      console.log(error)
+      this.setState({ isFetchingTicket: false });
+    })
   }
```
Comment (prashanthr) [inline src/components/coding-challenge/index.js]: @DSofter Can you point me to any online reference to the CORS issue you have been facing?
So we can't just use their create discount code API and then get the response?
http://teamtito.github.io/tito-api-docs/#creating-a-new-discount-code

Comment (prashanthr) [reply src/components/coding-challenge/index.js]: So even with the valid API Key and all the headers?
```
curl --request POST \
  --url 'https://api.tito.io/v2/an-account/awesome-conf/discount_codes' \
  --header 'Authorization: Token token=YOUR-API-KEY' \
  --header 'Accept: application/vnd.api+json' \
  --header 'Content-Type: application/json'
  --data '{"data":{"type":"discount-codes","attributes":{"code":"DISCOUNTDRACULA","discount_code_type":"PercentOffDiscountCode","value":"25.00"}}}'
```

Comment (DSofter) [reply src/components/coding-challenge/index.js]: @prashanthr If i do that locally with postman, it works. However if I do that locally within `localhost:8000`, it shows this error:
image

Comment (prashanthr) [reply src/components/coding-challenge/index.js]: I see. It does seem to be a documented issue.
https://github.com/gatsbyjs/gatsby/issues/2956
https://github.com/gatsbyjs/gatsby/pull/3299
- Try updating gatsby version
- Try using solutions mentioned in issue above

**Pull request merged**